setup.sh

```diff
@@ -1,21 +1,33 @@
 #!/bin/bash
 #
-# Sets up parameters for use with other scripts.  Should be run once.
+# Sets up parameters for use with other scripts.  Removes an instance of param.sh if there exists. 
 # Sample usage: bash setup.sh
-
+# Can be run for any number of times
+rm -r params.sh
 echo "First, choose the resource identifier that specifies your cluster resoure. We
 will set up this name in your ssh configuration, and use it to reference the resource (sherlock)."
 echo
 read -p "Resource identifier (default: sherlock) > "  RESOURCE
 RESOURCE=${RESOURCE:-sherlock}
+if [[ "${RESOURCE}" == "sherlock" ]]
+then
+   SHERLOCK=true
+   MACHINEPREFIX=${MACHINEPREFIX:-sh}
+else 
```

Comment:
Yep this works! If we do any hard coding of hostnames then it belongs here. The one condition you don't check for is the else - if the user is running from a cluster that is not sherlock nor farmshare. In this case you need to prompt the user for the shell prefix. Otherwise they will hit an error when they run the script.

Reply:
Added a new setup.sh file that takes in an additional input of the user machine prefix selection on 
ac87306.